screen-wake-lock/wakelock-supported-by-permissions-policy.html

```diff
@@ -1,5 +1,5 @@
 <!DOCTYPE html>
-<title>Test that wake-lock is advertised in the feature list</title>
+<title>Test that screen-wake-lock is advertised in the feature list</title>
```

Comment:
Note that document.featurePolicy is now document.permissionsPolicy.

Reply:
Ack. As I mentioned in the PR description, "wakelock-supported-by-permissions-policy.html continues to use document.featurePolicy because no engine currently implements document.permissionsPolicy" (not sure if the situation has changed since then though).

Reply:
Yeah, still appears to be the case.

Reply:
sorry I didn't see that int the commit message.